templates/install.html

```diff
@@ -0,0 +1,64 @@
+{% extends "_layouts/default.html" %}
+
+{% block meta_description %}Install the most streamlined linux VM management tool on Windows, Mac, or Linux. Multipass can create virtual machines in just seconds, and gives you the tools to run a local microcloud on your workstation.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1s_chvwjm3YaGQSmD_StXAwXElh9rsp-BBeFJWk0F7lk/edit{% endblock %}
+
+{% block content %}
+<section class="p-strip--suru">
+  <div class="row">
+    <div class="col-6">
+      <h1>Install Multipass</h1>
+      <p>Get an instant Ubuntu VM with a single command.  Multipass can launch and run virtual machines and configure them with <a href="https://cloud-init.io/" class="p-link--external p-link--inverted">cloud-init</a> like a public cloud. Prototype your cloud launches locally for free.</p>
+    </div>
+    <div class="col-6 u-hide--small u-align--center u-vertically-center">
+      <img src="https://assets.ubuntu.com/v1/0698ab2d-muiltipass-promo-header.png" style="height: 200px; width: 364px;" alt="Multipass"/>
```

Comment:
```suggestion
      <img src="https://assets.ubuntu.com/v1/0698ab2d-muiltipass-promo-header.png" style="height: 200px; width: 364px;" alt="Multipass logo"/>
```
The alt text should describe the image